test/css/component_specific_selectors_test.rb

```diff
@@ -16,10 +16,34 @@ class ComponentSpecificSelectorsTest < Minitest::Test
   IGNORED_SELECTORS = {
     :global => [/^\d/, ":"],
```

Comment:
I'm wondering if we should add a `[` character to ignored selectors? That would remove any that have attribute selectors on them like `[hidden]`. I'm just not sure it's feasible to test against those because presumably they're all event based like clicks

```suggestion
    :global => [/^\d/, ":", "["],
```

Reply:
+1

if we want to be a bit more specific, `/\[.*\]/` will look for the full attribute selector, like `[aria-label="foo"]`, but i don't know if it makes much difference (or any at all)

either one of these options should also take care of the `.ActionListItem[hidden]` issue noted above